packages/core/src/enhance/impl.js

```diff
@@ -1,39 +1,59 @@
 'use strict'
 
 import R from 'ramda'
-import { memoize, time } from '../impl/util'
+import { memoize } from '../impl/util'
 import * as data from '../data'
 import * as zip from '../zip'
 
-export function enhancer(config) {
-  const { registry, elementZipper } = config
+export function extract(config) {
+  const { registry, elementZipper, minNumOfArgs, maxNumOfArgs } = config
```

Comment:
This isn't really something that needs to be configurable, but rather part of the API.  Please remove.

Reply:
It's not really configurable. Only `enhance/index.js` knows about it: https://github.com/netceteragroup/girders-elements/pull/91/files#diff-ba2bb8dacdbda826717abca0faa4b195R55. I did it like this in order only one `extract` for both `extractContextBased` and `extractElementBased`. But I agree that it can be rethought.